embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testframework/actions/EmbracePayloadAssertionInterface.kt

```diff
@@ -94,7 +99,7 @@ internal class EmbracePayloadAssertionInterface(
      */
     internal fun getSessionEnvelopes(
         expectedSize: Int,
-        state: ApplicationState = ApplicationState.FOREGROUND
+        state: ApplicationState = ApplicationState.FOREGROUND,
```

Comment:
Unrelated to this changeset, but IMO I'd favour just making all of the formatting changes related to the new editor config in one big PR once we've landed the delivery layer project. Otherwise we'll have noise in all PRs for the next few months

Reply:
Yeah fair. It's the auto refactor that is touching of all these.

Should we just do one big reformat so we start at clean?

Reply:
Sounds good 👍